test/system/150-login.bats

```diff
@@ -209,6 +221,52 @@ EOF
     run_podman rmi $destname
 }
 
+@test "podman push and search ok with --tls-verify=true" {
```

Comment:
My one request would be, if at all possible, please don't copypaste. It should be possible to refactor the "podman push and search ok" test into a common-use function which accepts a conditional true/false for `tls`. (/me waves hands re: handling `--cert-dir`)